src/cljs/rems/application.cljs

```diff
@@ -765,7 +765,7 @@
                        (map dynamic-event->event (:dynamic-events app)))
         applicant-attributes (:applicant-attributes application)
         messages (remove nil?
-                         [(disabled-items-warning (:catalogue-items application)) ; NB: eval this here so we get nil or a warning
+                         [(when (editable? application)(disabled-items-warning (:catalogue-items application))) ; NB: eval this here so we get nil or a warning
```

Comment:
I'd prefer a line break at `)(`, but at least add a space :)

Reply:
should the condition actually be "when application is a draft"?

Reply:
I thought one can somehow modify returned application still, or does it create a new draft from the returned?

Reply:
oh right, I guess editable is the right condition then

Reply:
We are gonna dig a bit deeper tomorrow irl how this feature should be overall.